flutter/test/default_integrations_test.dart

```diff
@@ -75,6 +76,41 @@ void main() {
     expect(true, called);
   });
 
+  test('FlutterErrorIntegration captureEvent only called once', () async {
+    var numberOfDefaultCalls = 0;
+    final defaultError = (FlutterErrorDetails errorDetails) async {
+      numberOfDefaultCalls++;
+    };
+    FlutterError.onError = defaultError;
+
+    when(fixture.hub.captureEvent(captureAny))
+        .thenAnswer((_) => Future.value(SentryId.empty()));
+
+    final details = FlutterErrorDetails(exception: StateError('error'));
+
+    final errorIntegration = FlutterErrorIntegration();
+    errorIntegration.call(fixture.hub, fixture.options);
+    errorIntegration.call(fixture.hub, fixture.options);
+
+    FlutterError.reportError(details);
+
+    verify(await fixture.hub.captureEvent(captureAny)).called(1);
+    expect(numberOfDefaultCalls, 1);
+  });
+
+  test('FlutterErrorIntegration close restored default onError', () async {
+    final defaultOnError = (FlutterErrorDetails errorDetails) async {};
+    FlutterError.onError = defaultOnError;
+
+    final integrtion = FlutterErrorIntegration();
+    integrtion.call(fixture.hub, fixture.options);
+    expect(false, defaultOnError == FlutterError.onError);
+
+    integrtion.close();
+
+    expect(true, defaultOnError == FlutterError.onError);
```

Comment:
you can use `expect` along with `isTrue`, `isFalse`

Reply:
yeah last time I've asked someone else that worked on this repo, they preferred isTrue/isFalse, as I was not experienced in testing with Dart, we just decided going for it, so I'd rather keep `isTrue`/`isFalse` as we also use `isNull` or change everything else to equals, I'm fine either way, but it's nice to keep a standard pattern